<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateArchiveDialog</class>
 <widget class="QDialog" name="CreateArchiveDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create New Archive</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="outputGroupBox">
     <property name="title">
      <string>Output Archive</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLineEdit" name="outputFileLineEdit"/>
      </item>
      <item>
       <widget class="QPushButton" name="outputFileButton">
        <property name="text">
         <string>Browse...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="filesGroupBox">
     <property name="title">
      <string>Files to Archive</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="addFilesButton">
          <property name="text">
           <string>+ Add Files</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="addDirectoryButton">
          <property name="text">
           <string>+ Add Directory</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="removeButton">
          <property name="text">
           <string>Remove</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="clearButton">
          <property name="text">
           <string>Clear All</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QListWidget" name="fileListWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="fileCountLabel">
          <property name="text">
           <string>0 items</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="totalSizeLabel">
          <property name="text">
           <string>Total: 0 B</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="optionsGroupBox">
     <property name="title">
      <string>Options</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="compressCheckBox">
        <property name="text">
         <string>Enable Compression</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="compressionLevelCombo">
        <item>
         <property name="text">
          <string>Level 0 - No Compression</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 1 - Fastest</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 2-3 - Fast</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 6 - Default</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Level 9 - Best</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="encryptCheckBox">
        <property name="text">
         <string>Encrypt Archive</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateArchiveDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateArchiveDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>299</x>
     <y>474</y>
    </hint>
    <hint type="destinationlabel">
     <x>248</x>
     <y>374</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
